src/rust/lib/graph-editor/src/component/node/port.rs

```diff
@@ -263,7 +295,7 @@ impl ShapeViewDefinition for OutputPortView {
 }
 
 /// Helper trait that describes a `ShapeViewDefinition` with a port shape.
-pub trait PortShapeViewDefinition = ShapeViewDefinition<Shape=shape::Shape>;
+pub trait PortShapeViewDefinition = ShapeViewDefinition<Shape=shape::Shape> + Clone;
```

Comment:
Why this? And why not CloneRef?

Reply:
The additional bound is required for `WeakRef` etc. But `CloneRef` is enough.
Has been changd.